site/content/blog/2023-12-18-user-personas-poll/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Discovering Our Users - The ZAP User Personas Questionnaire"
+summary: >
+  Improving usability requires the identification of the type of users ZAP has
+images:
+- https://www.zaproxy.org/blog/2023-12-18-user-personas-poll/535a4e71-d6a6-40a1-8289-f31e8810b65b.png
+type: post
+tags:
+- blog
+- update
+date: "2023-12-19"
+authors:
+- yiannis
+---
+
+## Discovering Our Users - The ZAP User Personas Questionnaire
+
+The Zed Attack Proxy (ZAP) team is dedicated to making our tool the best it can be for our diverse user base. As the cybersecurity landscape evolves, so do the needs of our users. That's why we're excited to announce the **ZAP User Personas Questionnaire** — an initiative to better understand and cater to the unique requirements of everyone who uses ZAP.
+
+## **Why This Matters**
```

Comment:
Must admit I tend to use H3 ie `###` for blog post sub headings..

Reply:
Still looks like two and bold to me